### PR TITLE
[Housekeeping] Including pending fixes from Xamarin.Forms SRs

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change

Including pending fixes from Xamarin.Forms SRs.

https://github.com/xamarin/Xamarin.Forms/pull/15330
https://github.com/xamarin/Xamarin.Forms/pull/15299
https://github.com/xamarin/Xamarin.Forms/pull/15291
https://github.com/xamarin/Xamarin.Forms/pull/15369
https://github.com/xamarin/Xamarin.Forms/pull/1497

### CODE REVIEW

File: src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs [hunk 1]
```diff
@@ -124,7 +124,7 @@ void CheckForEmptySource()
 				// If we're going from empty to having stuff, it's possible that we've never actually measured
 				// a prototype cell and our itemSize or estimatedItemSize are wrong/unset
 				// So trigger a constraint update; if we need a measurement, that will make it happen
-				ItemsViewLayout.ConstrainTo(CollectionView.Bounds.Size);
+				ConstrainToItemsView();
```
Comment (jsuarezruiz) [inline src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs:127]: https://github.com/xamarin/Xamarin.Forms/pull/15289/files

File: src/Core/src/Handlers/Editor/EditorHandler.iOS.cs [hunk 2]
```diff
@@ -156,12 +156,20 @@ void OnStarted(object? sender, EventArgs eventArgs)
 
 		void OnEnded(object? sender, EventArgs eventArgs)
```
Comment (jsuarezruiz) [inline src/Core/src/Handlers/Editor/EditorHandler.iOS.cs:157]: From https://github.com/xamarin/Xamarin.Forms/pull/15369/files

File: src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs [hunk 3]
```diff
@@ -406,7 +406,14 @@ void UpdateFooter()
 			}
 
 			if (footer == null)
```
Comment (jsuarezruiz) [inline src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs:408]: From https://github.com/xamarin/Xamarin.Forms/pull/15306/files

File: src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs [hunk 4]
```diff
@@ -57,6 +57,12 @@ protected UITableViewRowAnimation ReloadSectionsAnimation
 			set { _dataSource.ReloadSectionsAnimation = value; }
 		}
 
+		[Internals.Preserve(Conditional = true)]
+		public ListViewRenderer(IntPtr handle)
```
Comment (jsuarezruiz) [inline src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs:61]: From https://github.com/xamarin/Xamarin.Forms/pull/14971/files

Comment (jsuarezruiz) [reply src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs:61]: Could fix this exception https://github.com/dotnet/maui/issues/7799

File: src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs (hunk 3 above)
Comment (jfversluis) [reply src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs:408]: Wait with this one, I might need to revert it as it causes a regression

File: src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs (hunk 1 above)
Comment (jfversluis) [reply src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs:127]: This one also reverted for now in Forms: https://github.com/xamarin/Xamarin.Forms/issues/15470

Comment (jsuarezruiz) [reply src/Controls/src/Core/Handlers/Items/iOS/ItemsViewController.cs:127]: Reverted

File: src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs (hunk 3 above)
Comment (jsuarezruiz) [reply src/Controls/src/Core/Compatibility/Handlers/ListView/Android/ListViewRenderer.cs]: Reverted.

**Comment by mattleibow** [comment]
From @PureWeen 

> Some of those fixes don't feel like they'd apply to MAUI 
>  - like dispatching setting the parent to null
>  - the focus change on editors
>  - Adding the ctor on ListViewRenderer
>  
> I feel like we'd want to get maui issues/repros put together for some of those to make sure they are still applicable and then fix with tests individually for SR if they are SR important

**Comment by jsuarezruiz** [comment]
To help focus and move faster, we're going to have fewer open PRs and focus on the ones we'll be working on in each release. This PR is not included in the next release so I close it for now. I will reopen it and adapt to changes later.

**Pull request closed without merging**